standalone/src/models/product-selection/product-of-selection/index.ts

```diff
@@ -0,0 +1,8 @@
+import { GraphqlModelBuilder } from './builders';
+import * as ProductOfSelectionPresets from './presets';
+export * from '../types';
+
+export const ProductOfSelectionGraphql = {
```

Comment:
Please leave a comment here explaining why we only have the GraphQL version of the model.